### ISSUE TITLE
Do we need a Toggle for Gravatar's Post Meta for Certain Specific Templates?

### ISSUE THREAD

**Issue by BurlesonBrad** [woothemes/storefront#383]
_....hold on to your eyeballs folks:_  **I can't think of a scenario where anyone would want all this meta on their homepage.**  _be thankful I didn't do a complete screenshot! my ugly mug (gravatar) actually gets echoed out *12* times)_

![sf-homepage-meta](https://cloud.githubusercontent.com/assets/6208757/14930071/e9d36d68-0e26-11e6-857a-11ce9ff8f6b5.png)
[http://i.imgur.com/96lA9OG.png](url)
__________________________________________________________
.....wait, _{thinking some more}_ 
 I can't think of a scenario where anyone would want this on a category page either
![sf-category-meta](https://cloud.githubusercontent.com/assets/6208757/14930347/ce70ab74-0e28-11e6-9118-4c9ba33a1d27.png)
[http://i.imgur.com/tyt5nQP.png](url)
___________________________________________________________
....hold on _{thinking some more}_ 
I don't see why this meta is necessary either.
![sf-single-meta](https://cloud.githubusercontent.com/assets/6208757/14930282/60a706d8-0e28-11e6-8edd-d35ca47aa84b.png)
___________________________________________________________
I'm running [2.0.0-beta-2](https://github.com/woothemes/storefront/tree/version/2.0.0-beta-2) and yes, this is a result of [Storefront Blog Customiser](https://www.woothemes.com/products/storefront-blog-customiser/). But does this bug need to be swatted here in Github or over in Zendesk?

Honestly, I can't help but wonder something......
While some of us might be able to toggle meta on and off | hook & unhook | and change templates... #### Would it benefit sales of [SF Blog Customiser](https://www.woothemes.com/products/storefront-blog-customiser/) if "Toggle Meta" was added and advertised as a feature?

_even if someone does **NOT** have disproportionately HUGE ears,_ I honestly can **not** see how in the world anyone would want/need the gravatar repeated over and over and over again.  

The [Storefront Blog Customiser](https://www.woothemes.com/products/storefront-blog-customiser/) needs some 2.0 love, and that's fine. 

But even w/out the Extension, can/could/should 2.0 include either a 'toggle' for gravatar ```is_front_page()``` ```is_home()``` ```is_archive ()``` _(....er.... *whatever* it's called)_  

Basically, if we've got a template layout that we know is going to repeat the meta information over and over and over and over and over again, should we toggle the ability to unhook/remove the gravatar? 

_no, I do *not* want to completely remove gravatar capacity alltogether!_  But Extension or Default, this gravatar hooplah isn't lookin' right.  _yes, it does need to stay in comments of course, hence *not* removing it completely_

**Comment by jameskoster** [comment]
This is totally dependent on the site in question. Obviously if _all_ posts on your blog are in the _same_ category and written by the _same_ author then yeah, the meta isn't gonna be all that helpful. Better to just have an 'author' widget in the sidebar or something.

Yeah, simple solution would be to have a meta toggle option in the Blog Customiser extension. I'll suggest that for the next release.
